app/models/profile.rb

```diff
@@ -9,6 +9,34 @@ class Profile < ApplicationRecord
 
   store_attribute :data, :custom_attributes, :json, default: {}
 
+  SPECIAL_FUNCTIONALITY_ATTRIBUTES = %w[
```

Comment:
Could you maybe elaborate a bit what the use-case for these constants will be? The idea was that once we're done generalizing profiles, new Forems will only get a handful of fields, so most of the ones here won't be available by default.

Reply:
My thought here is that these are fields which, if defined, are essentially reserved for special functionality. We don't need to require "email_public", _but_ if it does exist, it will have a certain special function... My thoughts are that we could do further work to make the special fields more general, but I think it's okay if "location" always means "location" and will trigger the icon indicating location. It's pretty global, and the stuff that isn't as global is still special if it exists.